react/features/base/config/reducer.js

```diff
@@ -157,43 +157,45 @@ function _translateLegacyConfig(oldValue: Object) {
 
     let newValue = oldValue;
 
-    // At the time of this writing lib-jitsi-meet will rely on config having a
-    // property with the name p2p and with a value of type Object.
-    if (typeof oldValue.p2p !== 'object') {
-        newValue = set(newValue, 'p2p', {});
-    }
-
-    /* eslint-disable indent */
+    const oldConfigToNewConfig = {
+        p2p: [
```

Comment:
I think it's time we drop this.

Reply:
OK. But this is totally out of the scope of this PR.